From 0b5f5d0be6a1c9aa2d63e72e6a979f46cfbcc866 Mon Sep 17 00:00:00 2001
From: Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
Date: Thu, 10 Jan 2013 09:20:22 +0100
Subject: [PATCH] Enable the Gateway and Source audio profiles by default.

Those can be disabled by the user in /etc/bluetooth/audio.conf if
necessary.

https://bugs.launchpad.net/ubuntu/+source/bluez/+bug/948613
---
 audio/a2dp.c    | 2 +-
 audio/manager.c | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/audio/a2dp.c b/audio/a2dp.c
index 404be53..6c0ab26 100644
--- a/audio/a2dp.c
+++ b/audio/a2dp.c
@@ -1444,7 +1444,7 @@ int a2dp_register(DBusConnection *conn, const bdaddr_t *src, GKeyFile *config)
 {
 	int sbc_srcs = 0, sbc_sinks = 0;
 	int mpeg12_srcs = 0, mpeg12_sinks = 0;
-	gboolean source = TRUE, sink = FALSE, socket = FALSE;
+	gboolean source = TRUE, sink = TRUE, socket = TRUE;
 	gboolean delay_reporting = FALSE;
 	char *str;
 	GError *err = NULL;
diff --git a/audio/manager.c b/audio/manager.c
index d442d1d..99e03bb 100644
--- a/audio/manager.c
+++ b/audio/manager.c
@@ -110,9 +110,9 @@ static GSList *devices = NULL;
 static struct enabled_interfaces enabled = {
 	.hfp		= TRUE,
 	.headset	= TRUE,
-	.gateway	= FALSE,
+	.gateway	= TRUE,
 	.sink		= TRUE,
-	.source		= FALSE,
+	.source		= TRUE,
 	.control	= TRUE,
 	.socket		= FALSE,
 	.media		= TRUE,
-- 
1.8.0.1

